@conda-forge/geos Any ideas on what the **linux-64** failure is here? Complains about `ctest` not recognizing `--exclude-regex` as an argument. That doesn't make sense to me - it shows CMake v3.30.1 is installed and that works fine on **osx-64** and **win-64**.

```cpp
...
[100%] Built target test_geos_unit
CMake Error: Unknown argument: --exclude-regex
CMake Error: Run 'ctest --help' for all supported options.
```

Otherwise, the **win-64** does show it now installs the `geos.pc` file, which is the point of the PR.

---

**NB:** Feel free to adjust the PR - I have nothing else planned for it.

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -17,6 +17,9 @@ if [[ "${OSX_ARCH}" = "x86_64" ]]; then
 fi
 
 if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
+    which ctest
+    ctest --version
+    ctest --help
     ctest --output-on-failure --exclude-regex ${CTEST_EXCLUDE}
 fi
 

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -11,16 +11,14 @@ cmake ${CMAKE_ARGS} \
 
 make -j${CPU_COUNT} ${VERBOSE_CM}
 
+CTEST_EXCLUDE=""
 if [[ "${OSX_ARCH}" = "x86_64" ]]; then
     # See https://github.com/libgeos/geos/issues/930
-    CTEST_EXCLUDE=unit-geom-Envelope
+    CTEST_EXCLUDE="--exclude-regex unit-geom-Envelope"
 fi
 
 if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
-    which ctest
-    ctest --version
-    ctest --help
-    ctest --output-on-failure --exclude-regex ${CTEST_EXCLUDE}
+    ctest --output-on-failure ${CTEST_EXCLUDE}
 fi
 
 make install -j${CPU_COUNT}

